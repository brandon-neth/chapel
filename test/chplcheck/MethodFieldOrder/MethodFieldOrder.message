/Users/brandonneth/chapel/test/chplcheck/MethodFieldOrder.chpl:31: node violates rule MethodsAfterFields
/Users/brandonneth/chapel/test/chplcheck/MethodFieldOrder.chpl:38: node violates rule MethodsAfterFields
/Users/brandonneth/chapel/test/chplcheck/MethodFieldOrder.chpl:4: node violates rule MethodsAfterFields
/Users/brandonneth/chapel/test/chplcheck/MethodFieldOrder.chpl:11: node violates rule MethodsAfterFields
