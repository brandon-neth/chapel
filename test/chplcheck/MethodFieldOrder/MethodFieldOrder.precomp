#!/bin/bash
$CHPL_HOME/tools/chapel-py/runChplcheck MethodFieldOrder.chpl > MethodFieldOrder.message