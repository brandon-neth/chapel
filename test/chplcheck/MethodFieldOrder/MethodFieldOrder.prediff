#!/bin/bash
while read expectedLine; do
    echo grep "$expectedLine" MethodFieldOrder.message | echo $? >> MethodFieldOrder.comp.out.tmp
done < MethodFieldOrder.expected