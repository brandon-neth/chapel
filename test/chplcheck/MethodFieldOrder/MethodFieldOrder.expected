MethodFieldOrder.chpl:31: node violates rule MethodsAfterFields
MethodFieldOrder.chpl:38: node violates rule MethodsAfterFields
MethodFieldOrder.chpl:4: node violates rule MethodsAfterFields
MethodFieldOrder.chpl:11: node violates rule MethodsAfterFields
